### ISSUE TITLE
hparams: Bug on displaying callable hyperparameters name

### ISSUE THREAD

**Issue by ThibzR** [lanpa/tensorboardX#579]
**Describe the bug**
A hyperparameter may not always be an `int`, `str`, `bool`, `float` or `None`. In my case, I have callables. For example, doing a hyperparameter optimization on final activations or even loss functions. The experiment runs fine (thanks to 9d267066) but on my `loss` column in `hparams` in Tensorboard, all fields under that are blank.

**Expected behavior**
If the param is a callable, take its name for proper display.

**Screenshots**
![image](https://user-images.githubusercontent.com/5993887/80744282-5633d400-8b16-11ea-9ff1-803337ec8087.png)
